charts/skupper-setup/Chart.yaml

```diff
@@ -1,4 +1,5 @@
 apiVersion: v2
 name: skupper-setup
 description:  Helm chart for setting up Skupper.
-version: 0.1.0
\ No newline at end of file
+version: 0.1.0
+appVersion: 2.0.0
```

Comment:
Thinking through the mechanics of releasing aloud here.

Today when we cut a release branch and update images.go with the next release tag. What happens in helm land?

I'd propose something like this:

**In "dev" branches - v2/main**
Chart version is chosen as a pre-release semver v2 - so for now 0.1.0-devel
Chart App Version is set to match the tag our CI produces - so v2-latest for now. Along with this, we plumb this field into the default values for skupper image tags.

**On release branch Create**
Drop the prerelease-label from the chart version: `0.1.0-devel` -> `0.1.0`.
Chart App version updated to next planned release: `v2-latest` -> `2.0.0`.

**Preparing for next patch on existing release branch**
Bump version `0.1.0` -> `0.1.1`
Bump app version `2.0.0` -> `2.0.1`

Reply:
good idea!